vault/resource_consul_secret_backend_role_test.go

```diff
@@ -0,0 +1,116 @@
+package vault
+
+import (
+	"encoding/base64"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/hashicorp/vault/api"
+)
+
+func TestResourceConsulSecretBackendRole(t *testing.T) {
```

Comment:
This doesn't pass for me currently. I get this: 
```
=== RUN   TestResourceConsulSecretBackendRole
--- FAIL: TestResourceConsulSecretBackendRole (0.02s)
    testing.go:434: Step 0 error: Error applying: 1 error(s) occurred:
        
        * vault_consul_secret_backend_role.test: 1 error(s) occurred:
        
        * vault_consul_secret_backend_role.test: error writing to Vault: Error making API request.
        
        URL: PUT http://localhost:8200/v1/consul/roles/test-5995672321196582423
        Code: 404. Errors:
        
        * no handler for route 'consul/roles/test-5995672321196582423'
FAIL

Process finished with exit code 1
```
It probably has something to do with this resource not being mounted?